src/webgl/3d_primitives.js

```diff
@@ -1532,4 +1532,54 @@ p5.RendererGL.prototype.curveVertex = function() {
   }
 };
 
+p5.RendererGL.prototype.image = function(
+  img,
+  sx,
+  sy,
+  sWidth,
+  sHeight,
+  dx,
+  dy,
+  dWidth,
+  dHeight
+) {
+  this._pInst.push();
+
```

Comment:
I think you should either take [the current `imageMode` into account ](https://p5js.org/reference/#/p5/imageMode) or put a [warning in here](https://github.com/processing/p5.js/blob/master/src/image/loading_displaying.js#L529) when the renderer is 3D.

Reply:
I still think there should be a warning [in imageMode](https://github.com/processing/p5.js/blob/master/src/image/loading_displaying.js#L529) when `renderer.isP3D`.